numpy/linalg/linalg.py

```diff
@@ -2639,6 +2639,13 @@ def multi_dot(arrays): return functools.reduce(np.dot, arrays)
         If the first argument is 1-D it is treated as row vector.
         If the last argument is 1-D it is treated as column vector.
         The other arguments must be 2-D.
+    out : ndarray, optional
+        Output argument. This must have the exact kind that would be returned
+        if it was not used. In particular, it must have the right type, must be
+        C-contiguous, and its dtype must be the dtype that would be returned
+        for `dot(a,b)`. This is a performance feature. Therefore, if these
+        conditions are not met, an exception is raised, instead of attempting
+        to be flexible.
```

Comment:
```suggestion
        to be flexible.
        
        .. versionadded:: 1.19.0
```

Reply:
@eric-wieser added in 8abeaba46a5fbf24b2c249cf5c9dbd8880f8231c